app/models/concerns/issues/views.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Issues::Views
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :views, dependent: :destroy
+  end
+
+  def view_by user
+    views.find_by(user_id: user.id)
```

Comment:
```suggestion
    views.where(user_id: user.id).exists?
```
Solo para que devuelva un booleano y lo dejamos bien "pregunta".

Reply:
Este te consulto franco tambien. Resulta que en la vista utilizamos el view que devuelve tambien para ver el created_at para informar cuando se vio:

`<% view ||= issue.view_by current_user %>

<% if view %>
  <%= icon 'fas', 'check', title: t('.viewed_at', on: l(view.created_at, format: :long)) %>`

Ahi que te parece lo pasamos a pregunta o lo dejamos asi para que podamos acceder a la fecha? Sino hago otro metodo que me devuelva esta fecha. Pero deberiamos confiar que el find_by devuelva la misma instancia.

Reply:
Ahh no me di cuenta que mostrábamos eso, dejalo sin tocar entonces, como estaba está bien, perdón por el amague =)

Reply:
perfecto franco, no pasa nadaaaa jajaja